integration-tests/src/test/java/org/zowe/apiml/integration/proxy/ServerSentEventsProxyTest.java

```diff
@@ -58,7 +58,7 @@ static void setup() throws SSLException {
     @Nested
     class WhenRoutingSession {
         @ParameterizedTest(name = "WhenRoutingSession.givenValidSsePaths_thenReturnEvents#message {0}")
-        @ValueSource(strings = {"/sse/v1/discoverableclient/events", SSE_ENDPOINT})
+        @ValueSource(strings = {DISCOVERABLE_SSE_EVENTS})
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed